backend/app/tests/api/routes/theme/test_post_theme_api.py

```diff
@@ -4,7 +4,7 @@
 from fastapi.testclient import TestClient
 
 from app.models.theme import MAX_NAME_LENGTH, MIN_NAME_LENGTH
```

Comment:
so this whole thing seems unrelated to the creating scheduled inquiries. Should this be a separate PR?

Reply:
This was related to a small refactor to avoid duplication.